functions/helpers.bash

```diff
@@ -498,3 +498,30 @@ is_wifi_connected() {
   if ! [[ -x $(command -v "comitup-cli") ]]; then return 1; fi
   if echo "q" | comitup-cli | grep -q 'State: CONNECTED'; then return 0; else return 1; fi
 }
+
+## Add dependency on ZRAM up: install/uninstall dependencies in zram-config.service
+## Argument 1 is "install" or "remove"
+## all remaining arguments are service names that zram must be available for to start
+##
+##    zram_dependency
+zram_dependency() {
+  local zramServiceConfig="/etc/systemd/system/zram-config.service"
+  local install=yes
+
+  zramServiceConfig="/tmp/z"
+  if ! [[ -f /etc/ztab ]]; then return 1; fi
+  if [[ "$1" == "install" ]]; then shift 1; fi
+  if [[ "$1" == "remove" ]]; then install=no; shift 1; fi
```

Comment:
What on earth is `shift 1` doing? I seem to be falling behind on my bash studies as of late.

Reply:
drops 1st argument i.e. moves (shifts) the rest 
Note you can call zram_dependency without either install or remove then the default (install) applies.

Reply:
OK, that makes sense.